### ISSUE TITLE
support channels for logrotate?

### ISSUE THREAD

**Issue by mzagrabe** [logrotate/logrotate#488]
Hello,

Thanks for writing logrotate!

Is the github issues the primary support channel for logrotate? Is there a mailing list or IRC channel?

Thanks for the info!

**Comment by kdudka** [comment]
Yes, GitHub is the support channel preferred by upstream developers of logrotate.
